recipes/wagtail-markdown

```diff
@@ -0,0 +1,45 @@
+{% set name = "wagtail-markdown" %}
+{% set version = "0.4.1" %}
+{% set sha256 = "89b24b04d874feb97adb0299b5fbc06ea29b5a835ef765b82846575a1d77f63f" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/torchbox/wagtail-markdown/archive/0.4.1.tar.gz
```

Comment:
The version is hard coded here. Guessing you meant to use Jinja?

Reply:
This is not on pypi, so I used a release from github.